src/integrationTest/java/uk/gov/hmcts/reform/cdapi/util/CommonDataApiPostgresqlContainer.java

```diff
@@ -6,7 +6,7 @@
 
 @Testcontainers
 public class CommonDataApiPostgresqlContainer extends PostgreSQLContainer<CommonDataApiPostgresqlContainer> {
-    private static final String IMAGE_VERSION = "postgres:11.1";
+    private static final String IMAGE_VERSION = "postgres:14";
```

Comment:
we can do this later, as we have to run the integrationTests against 11.1 for now?

Reply:
reverted